QUARKUS-1223.md

```diff
@@ -56,11 +56,14 @@ What is missing:
 ## Automated test development
 - Enable relevant Quickstart module in Quickstart acceptance job: https://github.com/quarkusio/quarkus-quickstarts/tree/main/hibernate-search-orm-elasticsearch-quickstart.
 - Implement new test scenario in QE test suite (include OpenShift tests) combining:
-  - Multiple persistence units, outbox polling and multi-tenancy (separate schema).
-  - RESTEasy Reactive.
```

Comment:
I would keep this line, add link to the blocker issue pls.

TestPlan is imho valid for this item, the fact that there is blocking issue is orthogonal to the plan.